content/SCALE/SCALETutorials/Network/ConfigureStaticRoutes.md

```diff
@@ -16,9 +16,9 @@ If you have a monitor and keyboard connected to the system you can use the [Cons
 
 If you need a static route to reach portions of the network, from the **Network** screen:
 
-1. Click **Add** in the **Static Routes** widget. The **Add Static Route** configuration screen displays. 
-   
-   {{< trueimage src="/images/SCALE/22.02/AddStaticRoute.png" alt="Add Static Routes" id="1: Add Static Routes" >}}
+1. Click **Add** in the **Static Routes** widget to show the **Add Static Route** screen.
```

Comment:
I've been standardizing on "open" rather than display or show, 
```suggestion
1. Click **Add** in the **Static Routes** widget to open the **Add Static Route** screen.
```